### DISCUSSION TITLE
Please, add customisable code snippets 🙏🙏🙏

### DISCUSSION THREAD

**Discussion by GrigoriiPrudnikov** [zed-industries/zed#14584] [Feature Requests]
I want to have ability to create my own code snippets. For example:

`rfce` (react functional component export) + `tab` 

will create this block of code

```jsx
export function FileName() {
    return <div>Filename</div>
}
```

**Comment by 0x2CA** [comment]
Hi, now that snippet is supported, it is possible to create files to customize the.
[Add language-agnostic snippets #13253](#13253)
[Snippets: Move snippets into the core of editor #13937](#13937)

**Reply by JosephTLyons** [reply]
There is also a tiny bit of documentation here that gives some JS examples for how to set these up.

https://zed.dev/docs/snippets
